packages/wallet_kit/lib/widgets/wallet_list.dart

```diff
@@ -206,10 +206,9 @@ class WalletCell extends HookConsumerWidget {
                   label: const Text('Add account'),
                   onPressed: () async {
                     ref.read(walletsProvider.notifier).addAccount(
-                          walletId: wallet.id,
-                          getPassword: () => showPasswordModal(context),
-                        );
-                    Navigator.of(context).pop();
+                        walletId: wallet.id,
+                        getPassword: () => showPasswordModal(context),
+                        context: context);
```

Comment:
If you remove the `Navigator.of(context).pop()`, the wallet popup will still be displayed after successfully adding an account.

Reply:
Fixed